dockerfiles/Dockerfile.apisix.rpm

```diff
@@ -7,16 +7,24 @@ ARG apisix_repo="https://github.com/apache/apisix"
 FROM ${image_base}:${image_tag}
 
 RUN set -x \
-    && yum -y install wget tar gcc automake autoconf libtool make curl git which unzip \
+    && yum install pcre-devel openssl-devel gcc curl wget tar automake autoconf libtool make git which unzip
+    && wget https://openresty.org/download/openresty-1.17.8.2.tar.gz
```

Comment:
We should install OpenResty from the official repo. It is OK to use OpenResty 1.19 in the build. I think the output can work with OpenResty 1.17 as they use the same OpenSSL library and compatible Lua API.

Reply:
fixed.